src/main/java/task/Todo.java

```diff
@@ -0,0 +1,28 @@
+package task;
+
+/**
+ * Represents a type of Task - Todo.
+ * A generic task that supports any type of text as a description.
+ * Distinguishes itself from an Event or Deadline.
+ */
+public class Todo extends Task {
+
+    /**
+     * Class constructor.
+     *
+     * @param description Description of todo
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Returns todo details.
```

Comment:
Maybe todo can be capitalized to ToDo to show that it returns a instance of a class?